internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -10,6 +10,10 @@ import (
 	"regexp"
 	"strings"
 
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
```

Comment:
Same here, somehow these got shifted up and isolated by empty lines above and below haha (the above makes sense)